File: comms/push-notifications/users/list-users/list-users.java
package com.example.usage;

import com.twilio.comms.TwilioCommsClient;
import com.twilio.comms.resources.pushnotifications.users.requests.UsersListRequest;

public class Example {
    public static void main(String[] args) {
        TwilioCommsClient client = TwilioCommsClient
            .builder()
            .credentials("<username>", "<password>")
            .build();

        client.pushNotifications().users().list(
            UsersListRequest
                .builder()
                .pageToken("pageToken")
                .pageSize(50)
                .build()
        );
    }
}

File: comms/push-notifications/users/list-users/list-users.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "<username>",
        authToken: "<password>",
    });
    await client.pushNotifications.users.list({
        pageToken: "pageToken",
        pageSize: 50,
    });
}
main();
